.github/workflows/industrial_ci_action.yml

```
@@ -0,0 +1,101 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
+# For troubleshooting, see readme (https://github.com/ros-industrial/industrial_ci/blob/master/README.rst)
+
+name: BuildAndTest
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+      - "[kmn]*-devel"
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - IMAGE: 'melodic-ci'
+            CCOV: true
+          - IMAGE: 'melodic-ci-shadow-fixed'
+            IKFAST_TEST: true
+            CATKIN_LINT: true
+    env:
+      DOCKER_IMAGE: moveit/moveit:${{ matrix.env.IMAGE }}
+      UPSTREAM_WORKSPACE: .ci.upstream.rosinstall
```

There is only a single upstream dependency. Instead of cluttering the filesystem with yet another file (`.ci.upstream.rosinstall`), we can define the github repo directly:
```suggestion
      UPSTREAM_WORKSPACE: github:ros-planning/moveit_resources#melodic-devel
```